### PR TITLE
Firefox 131 adds preliminary support for position-area

### PR DESCRIPTION
**Pull request by dletorey**
#### Summary

- Added release for `position-area` for firefox 131

#### Test results and supporting details

Tested in Firefox Nightly and the value parses in devTools, this feature also requires `anchor-name` and position-anchor` properties for it to work.

#### Related issues

- [Firefox 131 Release note PR](https://github.com/mdn/content/pull/35916)

### CODE REVIEW

File: css/properties/position-area.json [hunk 1]
```diff
@@ -21,8 +21,7 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": false,
-              "impl_url": "https://bugzil.la/1838746"
+              "version_added": "131"
```
Comment (caugner) [inline css/properties/position-area.json:24]:
```suggestion
              "version_added": "131",
              "partial_implementation": true,
              "notes": "While the CSS property is parsed and accepted, it does not have any effect."
```

File: css/properties/position-area.json [hunk 2]
```diff
@@ -40,7 +39,7 @@
             "webview_ios": "mirror"
           },
           "status": {
-            "experimental": true,
+            "experimental": false,
```
Comment (caugner) [inline css/properties/position-area.json:42]: Although the data guidelines [don't specify this](https://github.com/mdn/browser-compat-data/blob/a244bf0b3ffe1f897c4e3bafd0e62ce211f694f0/docs/data-guidelines/index.md?plain=1#L122-L133), we should probably consider a partial implementation as no support for the experimental setting.

File: css/properties/position-area.json [hunk 3]
```diff
@@ -58,8 +57,7 @@
               "chrome_android": "mirror",
               "edge": "mirror",
               "firefox": {
-                "version_added": false,
-                "impl_url": "https://bugzil.la/1838746"
+                "version_added": "131"
```
Comment (caugner) [inline css/properties/position-area.json:60]: I would consider all values as not supported, as they have no effect.

**Review by caugner — CHANGES_REQUESTED** [review]

File: css/properties/position-area.json (hunk 1 above)
Comment (dletorey) [reply css/properties/position-area.json:24]: Technically it is a full implementation of `position-area`.
I shall add the reason to the note
> "The property is parsed and accepted, but it does not have any effect, as it also needs the implementation of `anchor-name` and `position-anchor`."

File: css/properties/position-area.json (hunk 3 above)
Comment (dletorey) [reply css/properties/position-area.json:60]: But the `position-area` property does parse so technically is supported.

File: css/properties/position-area.json [hunk 4]
```diff
@@ -21,7 +21,8 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "131"
+              "version_added": "131",
+              "notes": "The property is parsed and accepted, although it does not have any effect as it also needs the implementation of <a href='https://developer.mozilla.org/docs/Web/CSS/anchor-name'>`anchor-name`</a> and <a href='https://developer.mozilla.org/docs/Web/CSS/position-anchor'>`position-anchor`</a>."
```
Comment (caugner) [inline css/properties/position-area.json:25]: This sounds like a partial implementation?

Do I understand this right: The property only has an effect when _combined_ with anchor-name and (or?) position-anchor?

Comment (dletorey) [reply css/properties/position-area.json:26]: added

Comment (caugner) [reply css/properties/position-area.json:26]: @dletorey Thanks for adding the partial option.

Could we make the note clearer? Something along these lines:

```suggestion
              "notes": "The property is parsed and accepted, but it has no effect alone, and both <a href='https://developer.mozilla.org/docs/Web/CSS/anchor-name'>`anchor-name`</a> and <a href='https://developer.mozilla.org/docs/Web/CSS/position-anchor'>`position-anchor`</a> are not yet supported."
```

**Review by caugner — APPROVED** [review]

**Pull request merged**